hadoop-ozone/dev-support/checks/hadolint.sh

```diff
@@ -17,25 +17,30 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 cd "$DIR/../../.." || exit 1
 REPO_DIR="$DIR/../../.."
 
-ERROR=0
-
+REPORT_DIR=${OUTPUT_DIR:-"$DIR/../../../target/hadolint"}
+mkdir -p "$REPORT_DIR"
+REPORT_FILE="$REPORT_DIR/summary.txt"
 
 for Dockerfile in $(find "$REPO_DIR/hadoop-ozone" "$REPO_DIR/hadoop-hdds" -name Dockerfile | sort); do
-  echo "Checking $Dockerfile"
+  echo "Checking $Dockerfile" | tee -a "$REPORT_FILE"
```

Comment:
This would cause exit code 1, because even without any errors, at least `Checking ...` would be present in the report file, hence `-s` evaluates to true.  I think we can remove this line, as hadolint's error messages include the input file's path.

Reply:
Sure. If we are removing the 'Cheking ...', I was wondering if we can remove the spaceline too.